doc/website/.pages

```diff
@@ -1,6 +1,10 @@
 nav:
   - index.md
   - getting-started
-  - API-reference
+  - concepts
+  - examples
+  - design
```

Comment:
same same

Reply:
At the moment just an empty document with a `@todo` is in in `doc/website/design`. If we don't manage to revise the design docs during the release validation, we can remove it.